lang/python/python3-kconfiglib/Makefile

```diff
@@ -0,0 +1,41 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python3-kconfiglib
+PKG_RELEASE:=1
+PKG_VERSION:=13.1.1
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/ulfalizer/Kconfiglib.git
+PKG_SOURCE_VERSION:=bf01114a0ce09b891ed04609ea7e9cc6f3ef3df2
+PKG_BUILD_DIR:=$(BUILD_DIR)/python3-kconfiglib-$(PKG_VERSION)
+PKG_LICENSE:=ISC
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=John Crispin <john@phrozen.org>
+HOST_BUILD_DEPENDS:=python3/host
```

Comment:
This looks bad. It's ugly. Please take a look at how other packages in the Python folder looks like.
E.g. https://github.com/openwrt/packages/blob/master/lang/python/python-attrs/Makefile

Reply:
@BKPepe let me rephrase that for you "Hi kindly consider to making this patch aligned with how other packages do it. take a look at <...> for a referecne"